test/tests/outputs/png/adv_diff_reaction_test.i

```diff
@@ -0,0 +1,124 @@
+[Mesh]
```

Comment:
Great! Now you need to add the "tests" file so that these actually get run. I'm going to suggest a "CheckFiles" test but we might be able to do better. 

@aeslaughter - Can we do a png difference or is that only available within the python testing infrastructure?